src/package/packageConvert.ts

```diff
@@ -264,6 +276,40 @@ export async function createPackageVersionCreateRequest(
   return createRequestObject(packageId, context, packageVersTmpRoot, packageVersBlobZipFile);
 }
 
+function buildPackageDescriptorJson(args: {
+  packageId: string;
+  base?: Partial<PackageDescriptorJson>;
+  project?: SfProject;
```

Comment:
I think project is required for the convert command to run so this could probably be simplified a bit.